source/features/linkify-commit-sha.js

```diff
@@ -0,0 +1,13 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/utils';
+
+export default function () {
+	let commitUrl = location.pathname.replace(/\/$/, '');
```

Comment:
Why are you dropping the trailing slash? Doesn't seem needed

Reply:
It's copied from here for consistency:

https://github.com/sindresorhus/refined-github/blob/7e689e11620a797bdb693d8dc700f542da6a76fd/source/features/add-patch-diff-links.js#L6

Should the function be moved to `libs` and reused? If so, do you recommend where to move it?

Reply:
That’s required because to the url we append `.diff` but here it isn’t. Also this can probably be inclined in the JSX

Reply:
👍 , code simplified in 2985e9d.